docs/src/main/sphinx/admin.md

```diff
@@ -36,6 +45,11 @@ admin/event-listeners-mysql
 admin/event-listeners-openlineage
 ```
 
+Unrelated to event listeners, the coordinator stores information about recent
+queries in memory for usage by the [](/admin/web-interface) - see also
+`query.max-history` and `query.min-expire-age` in
+[](/admin/properties-query-management).
```

Comment:
Because its only technically unrelated from the implementation .. for users it is very much related .. since they ask .. how can I get more query history .. first in the UI .. and then in general .. and the answer for "in general" is event listeners

Reply:
Or .. I have more info in the event listener backend now .. but what if I need more in the UI as well